### DISCUSSION TITLE
driver.execute(mobile: deep-link)

### DISCUSSION THREAD

**Discussion by dayafigueroa** [appium/appium#16518] [Q&A]
Hi... I have implemented deeplink for android using 
driver.execute('mobile:deepLink', {
		url: '<url>',
		package: '<package>',
	});

I started my appium server with --relaxed-security
In the appium logs i see the command getting generated and running. I also see a 200 (indicating success), however nothing happens on the simulator. 

<img width="940" alt="image" src="https://user-images.githubusercontent.com/4635367/156434386-00082c90-d1d9-452a-a116-8c849aa76144.png">

If i run the command on a different terminal tab it does what i'm expecting it to do. 
<img width="952" alt="image" src="https://user-images.githubusercontent.com/4635367/156433692-12e9a573-db2c-44fb-8169-cfe111724a48.png">

I'm using  "webdriverio": "^7.7.3"

please advice...

**Comment by KazuCocoa** [comment]
What about the logcat? Perhaps logcat has something about the adb shell call that the device received
